versionedRouter.js

```diff
@@ -557,6 +557,46 @@ if (transformerTestModeEnabled) {
       ctx.status = 400;
     }
   });
+  // code: string;
+  // codeVersion: string;
+  // language: string;
+  // id?: string;
+  // name?: string;
+  // testWithPublish?: boolean;
+
+  router.post("/transformation/testhandler", async ctx => {
```

Comment:
Why a new one instead of using the existing custom transform route?

Reply:
Now we have two routes. one to setup infra and other to test function.
This is to not worry about different flows in same endpoint.
`/customTransform` endpoint is called from server while processing events
`/transformation/sethandle` endpoint to init infra needed for invocation, called from config-backend
`/transformation/test` endpoint to test code, called from config-backend